data/scripts/lib/xml.lua

```diff
@@ -0,0 +1,18 @@
+local boolTrue = {"1", "y", "Y", "t", "T"}
+
+function tobool(str)
+    return str and table.contains(boolTrue, str[0])
+end
+
+function XMLNode.children(self)
+    local node = self:firstChild()
+    return function()
+        if not node then
+            return nil
+        end
+
+        local prevNode = node
+        node = node:nextSibling()
+        return prevNode
+    end
+end
```

Comment:
In case anyone wonders, this implements the iterator protocol (which is super weird in Lua, but w/e) so one can use:
```lua
for child in node:children() do
  ...
end
```